cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -48,7 +49,7 @@ Model::Model(std::shared_ptr<wgpu::Device> device, const Model::Options& options
   this->pipeline = deviceValue.CreateRenderPipeline(&descriptor);
 }
 
-void Model::setAttributes(const std::shared_ptr<garrow::Table>& attributes) { this->_attributes = attributes; }
+void Model::setAttributes(const std::shared_ptr<garrow::Table>& attributes) { this->_attributeTable = attributes; }
```

Comment:
Reminder: We do need two tables, one for instanced attributes and another for the "primitive" attributes. So we will need two functions here.

For LineLayer and ScatterplotLayer, we will need this to be the instanced table.